src/components/tasks/card/ProgressBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { ProgressBarProps } from '@/interfaces/task.type';
+import { FC } from 'react';
+import ProgressSlider from './ProgressSlider';
+import ProgressIndicator from './ProgressIndicator';
+
+const HandleProgressbar: FC<ProgressBarProps> = ({
+    progress,
+    progressValue,
+    percentCompleted,
+    handleProgressChange,
+    debounceSlider,
+    startedOn,
+    endsOn,
+}) => {
+    if (progress) {
+        return (
+            <>
+                <ProgressSlider
+                    value={progressValue}
+                    debounceSlider={debounceSlider}
+                    handleProgressChange={handleProgressChange}
+                />
+                <span>{progressValue}%</span>
+            </>
+        );
+    } else {
```

Comment:
- Should we do a default return instead of using `else`? 
The code will be less nested.

Reply:
fixed